src/components/caroulsel.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+/* package: https://www.npmjs.com/package/react-alice-carousel */
+import AliceCarousel from 'react-alice-carousel';
+import 'react-alice-carousel/lib/alice-carousel.css';
+
+const handleDragStart = (e) => e.preventDefault();
+
+const items = [
+  <div className='quickstart' onDragStart={handleDragStart}>
+    <a href='./mobile-apps/automated-testing/appium/quickstart/' target='_self'>
```

Comment:
I don't think you need the `.` at the beginning? @spider-sauce -- can you confirm?

Reply:
yes, you don't need the `.`, the root in this case is the root of the site, which is the `docs` root folder.